mgr.go

```diff
@@ -327,16 +325,19 @@ func (mgr *SysboxMgr) unregister(id string) error {
 	}
 
 	// revert mount prep actions
-	mgr.mntPrepLock.Lock()
 	for _, revInfo := range info.mntPrepRev {
 		if revInfo.chown {
+			logrus.Infof("reverting chown on %s to %d:%d for %s", revInfo.path, revInfo.origUid, revInfo.origGid, id)
```

Comment:
I put them as Info logs because the chown operation can be slow still, and thus may still be problematic. If we hit the problem, I wanted some indication of what's happening in the logs.